src/libraries/Microsoft.PowerFx.Core/Entities/External/IExternalDataSource.cs

```diff
@@ -19,6 +19,8 @@ internal interface IExternalDataSource : IExternalEntity, IExternalPageableSymbo
 
         bool RequiresAsync { get; }
 
+        bool IsWritable { get; }
```

Comment:
>bool IsWritable { get; } [](http://example.com/codeflow?start=8&length=24)

Originally pushed by [https://github.com/microsoft/Power-Fx/pull/2246](https://github.com/microsoft/Power-Fx/pull/2246). Including it here since it was meant to fix things in this PR anyway. This does not break PA.